<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_Set</class>
 <widget class="QWidget" name="User_Set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>客户设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>700</width>
      <height>340</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>160</width>
       <height>320</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>自检</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>配置设置</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>其他设置</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>470</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
  min-height: 28px; min-width: 40px;
/*  background-color: rgb(0, 170,0);*/
  background-color: rgb(12, 148, 214);

}

QComboBox QAbstractItemView::item {
  min-height: 35px; min-width: 60px;
 /* background-color: rgb(0, 170,0);
  background-color: rgb(75, 190, 155);*/

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>460</width>
        <height>320</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>460</width>
          <height>320</height>
         </rect>
        </property>
        <widget class="QPushButton" name="Year_Detection">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>170</y>
           <width>200</width>
           <height>70</height>
          </rect>
         </property>
         <property name="text">
          <string>手动年检</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Month_Detection">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>200</width>
           <height>70</height>
          </rect>
         </property>
         <property name="text">
          <string>手动月检</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>451</width>
          <height>311</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>230</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>配电支路数:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="Distribution_branches_number">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>230</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_ATS">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>120</y>
           <width>161</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ATS配置:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>配电配置:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_Distribution">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>190</y>
           <width>161</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_protectenable">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>161</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>主电过载保护:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_outputalarmenable">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>60</y>
           <width>161</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>输出开路告警:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SWITCH">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>270</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>COM3</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_ATS">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>160</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>COM4</string>
         </property>
        </widget>
        <widget class="QPushButton" name="setAlarm">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>230</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>告警设置</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="btn_restore">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string>恢复出厂设置</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_reset">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_restore</tabstop>
  <tabstop>Distribution_branches_number</tabstop>
  <tabstop>comboBox_ATS</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>Year_Detection</tabstop>
  <tabstop>Month_Detection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
